notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -55,10 +55,10 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        value=model_ids[0],
</pre><p>could you please change index by default on llama or mpt as we have some issues with red-pijama (that is not solved yet) and this sample is to so great for demonstrating int4</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1428/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Made MPT the default option because running llama is quire complicated requiring to gain access rights.</p>